addons/mail/static/tests/mobile/composer_tests.js

```diff
@@ -0,0 +1,27 @@
+/* @odoo-module */
+
+import { startServer } from "@bus/../tests/helpers/mock_python_environment";
+
+import { start } from "@mail/../tests/helpers/test_utils";
+import { triggerHotkey, patchWithCleanup } from "@web/../tests/helpers/utils";
+import { contains, insertText } from "@web/../tests/utils";
+import { browser } from "@web/core/browser/browser";
+
+QUnit.module("mobile composer");
+
+QUnit.test(
+    'Mobile: do not send message with enter key',
+    async () => {
+        const pyEnv = await startServer();
+        const channelId = pyEnv["discuss.channel"].create({ name: "General" });
+        // patching navigator to fake a mobile device
+        patchWithCleanup(browser.navigator, {
+            userAgent: "Chrome/0.0.0 Android (OdooMobile; Linux; Android 13; Odoo TestSuite)",
+        });
+        const { openDiscuss } = await start();
+        openDiscuss(channelId);
+        await insertText(".o-mail-Composer-input", "Test");
```

Comment:
IIRC `openDiscuss(channelId)` doesn't open the right thread immediately in mobile in version 17.0
Should instead `openDiscuss()` without param and open the thread by hand with `await click()`.